tsfile/src/main/java/org/apache/iotdb/tsfile/file/footer/ChunkGroupFooter.java

```diff
@@ -140,7 +130,8 @@ public int serializeTo(OutputStream outputStream) throws IOException {
     length += ReadWriteIOUtils.write(deviceID, outputStream);
     length += ReadWriteIOUtils.write(dataSize, outputStream);
     length += ReadWriteIOUtils.write(numberOfChunks, outputStream);
-    assert length == getSerializedSize();
+    boolean isEqual = (length == getSerializedSize());
+    assert isEqual;
```

Comment:
I don't think using assertion is a good idea, maybe you can convert it to an exception.

Reply:
ok